src/__tests__/is-safari.test.ts

```diff
@@ -0,0 +1,65 @@
+import isSafari from "../is-safari";
+
+const AGENTS: {
+  [key: string]: string;
+} = require("./helpers/user-agents.json");
+
+describe("isSafari", () => {
+  it("returns true when safari", () => {
+    expect(isSafari(AGENTS.macSafari7_0_2)).toBe(true);
+    expect(isSafari(AGENTS.pcSafari5_1)).toBe(true);
+  });
+
+  it("returns false when iOS safari", () => {
+    expect(isSafari(AGENTS.iPhone_9_3_1Safari)).toBe(false);
+    expect(isSafari(AGENTS.iPhone_3_2Safari)).toBe(false);
+    expect(isSafari(AGENTS.iPad3_2Safari)).toBe(false);
+    expect(isSafari(AGENTS.iPad5_1Safari)).toBe(false);
+    expect(isSafari(AGENTS.iPad9_3Safari)).toBe(false);
+    expect(isSafari(AGENTS.iPodSafari)).toBe(false);
+  });
+
+  it("returns false for iOS Facebook", () => {
+    expect(isSafari(AGENTS.iPhoneFacebookWebview)).toBe(false);
+  });
+
+  it("returns false when chrome", () => {
```

Comment:
when Chrome... desktop? mobile? We may want to make these test descriptions more clear